openwisp_controller/connection/api/serializers.py

```diff
@@ -31,6 +32,21 @@ class CommandSerializer(ValidatedDeviceFieldSerializer):
         pk_field=serializers.UUIDField(format='hex_verbose'),
     )
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # show only connections and command types available for the device
```

Comment:
Hi @DragnEmperor 
Can we have a test to also ensure that only connections related to the device are available? Not able to find a similar one is code